shell.js

```diff
@@ -162,6 +162,35 @@ module.exports = function() {
     }
   };
 
+  var profileProcess = function(args, system, cb) {
+    if (!_runner.isProcessRunning(args.process)) {
```

Comment:
I've implemented this in fuge-runner, since we've looked up the container in fuge-runner already and seems to be a more natural place (according to surrounding code context) to implement this kind of check